CHANGES.md

```diff
@@ -3,6 +3,14 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.462
+- Updated model version to v0.0.414
+  - Added node pool endpoint resources for aro_hcp.
```

Comment:
Nitpick: n this same PR in some places we end a list element with the `.` character and not in some others. We should be consistent.

For the latest few releases we haven't been using `.` so I propose the same here.

As a side note, taking a look at the file itself in general we are not being consistent about that unfortunately

Reply:
Oh, good catch. Fix it.